infra/src/main/kotlin/fi/fta/geoviite/infra/publication/PublicationDao.kt

```diff
@@ -2376,48 +2374,88 @@ class PublicationDao(
         return fetchPublicationIdByUuid(uuid)?.let(::getPublication)
     }
 
-    fun fetchPublishedLocationTracksAfterMoment(
+    fun fetchPublishedLocationTrackBetween(
```

Comment:
Ulospäin erilliset funkkarit yhden vs kaikkien muutosten haulle. Siitä ohjaus internaliin joka hoitaa sql-puolen molemmille.